Tests/EventDispatcherTests.swift

```diff
@@ -0,0 +1,198 @@
+import Quick
+import Nimble
+import Clappr
+
+class EventDispatcherTests: QuickSpec {
+
+    override func spec() {
+        describe("EventDispatcher") {
```

Comment:
According to the Project conventions, we should add the dot(.) prefix on the Class Name.

``context(".stopListening")``